Check and balances.md

```diff
@@ -6,16 +6,35 @@ permalink: /checks_and_balances/
 
 # Check and balances : sdg.market
 
-## Selling project parts wihtout owning some
+## Selling project parts without owning some
 In this case. Delijaj is the seller and Nyomi the buyer.
-While testing the trading system on the 3<sup>rd</sup> auction, a seller decided to trad some parts of the project. 
-However, she did not own any part of it but the website let her do it anyway. The parts were at 50 token. A buyer 
+While testing the trading system on the 3<sup>rd</sup> auction, a seller decided to trad some parts of the project.
+However, she did not own any part of it but the website let her do it anyway. The parts were at 50 token. A buyer
 decided to acquire those parts. The exchange was made and the seller finished with a -50 on the project (like a debt).
-This shouldn't happend since the parts must be only managed by the contributors and nobody else.
-
-## Sponsor/Inversor 
-As the website is today, the trade is at its basic stage. To improve it, we can put a system of sponsor/investor.
-Indeed if we look at the projects, some people will have fantastic ideas but not the means to do it. So to help developping 
-the market and the motivation to creat a projet. In the case that a person doesn't want to continue the project, for any reasons,
-she could sell it in the platforme to keep it going.
-It will creat a real market and viability of the projects.
+The admin of the project should have the power to say if he wants or not such things.
+
+## Licence
+I don’t see the point on creating a new licence. I feel like we should use what we already have since creating such a licence
+would be assembling existing licences. (and some project would have similar stuff).
+“provisional financial attribution based on share ownership, which itself is based on the contribution & trading trails
+over the whole history of the project.” This is not in the good place for me, it should be shown at the contributor’s
+profile and thus on how we manage the profiles.
+
```

Comment:
It makes sense if the licensing is linked to financing... I agree with you that the need for a new license is not the most pressing current issue. But... financing models are ver linked to the licensing of projects. It is very well known that it is way easier to monetize proprietary products than it is "free " (as in free speech) products that is why it seems tempting to create a new license... It might be the case eventually bt there is already a lot of work being done in that area with licenses like AGPL and SSPL for example.